styles/mobile.css

```diff
@@ -1,58 +1,72 @@
-@media screen and (max-width: 768px)
-       {
-           footer hr
-           {
-               display: none;}
-           
-           .paperclip
-                    {
-               display: none;}
-           
-           .mugshot 
-           { float: none;
-            margin-left: 0;
-            margin-right: 0;}
-            
-            main 
-              {
-                  display: block;
-               }
-               .row-span-2{
-       grid-row: span 2;
-      }
-    
-      .col-span-2{
-       grid-column: span 2;
-       }
-       header nav ul {
-           display: flex;
-           flex-direction: column;
-           text-align: center;
-       }
-       header a:hover
-       {
-           border: 0;
-           text-decoration:  underline;
-       }
-       .burger,
-.burger::before,
-.burger::after {
-   width: 24px;            
-border-top: 4px solid black;
-   }
-.burger::before,
-.burger::after {
-  content: "";
-  display: block;
-  position: absolute;}
-.burger {
-  height: 16px;
-  cursor: pointer;
-  position: relative;
-  margin-left: auto;
-  margin-right: auto;
- }
-.burger::before{top: 4px; }  
-  .burger::after{top: 12px; } 

-  }
-   
\ No newline at end of file
+@media screen and (max-width: 768px) {
+    footer hr {
+        display: none;
+    }
+
+    .paperclip {
+        display: none;
+    }
+
+    .mugshot {
+        float: none;
+        margin-left: 0;
+        margin-right: 0;
+    }
+
+    main {
+        display: block;
+    }
+
+    .row-span-2.col-span-2 {
+        grid-row: span 1;
+        grid-column: span 1;
+    }
```

Comment:
Note this rule in comparison to what you had before.  This rule applies only to those element with BOTH the classes.  Also, note that we are only spanning 1 column/row as requested by the lab